### DISCUSSION TITLE
Common tags OpenTelemetry

### DISCUSSION THREAD

**Discussion by lbroderi-li** [open-telemetry/community#1667] [Community Discussions]
How can you add common tags to all metrics with OpenTelemetry? I am using micrometer and have setup an OpenTelemetryMeterRegistry using the .config().commonTags(commonTags) addon. However, these tags are not appearing on my metrics, so wondering if there is a different approach recommended?

**Comment by trask** [comment]
hi @lbroderi-li! I'd suggest asking in https://github.com/open-telemetry/opentelemetry-java-instrumentation
